.env.example

```diff
@@ -3,6 +3,5 @@ TWITTER_CONSUMER_SECRET=XXXXXXXXXXXXX
 TWITTER_ACCESS_TOKEN=XXXXXXXXXXXXX
 TWITTER_ACCESS_SECRET=XXXXXXXXXXXXX
 REDIS_URL=redis://localhost:6379/5
-DATABASE_URL=postgresql://postgres:12345@localhost:5432/mame-challenge_development
 PORT=3000
 RAILS_ENV=development
```

Comment:
This line might also need to be removed, not sure what the implication though, but copying `.env.example` as `.env.test` and having `RAILS_ENV=development` on test environment doesn't sounds right

Reply:
👍 Yep, I think it should be safe to remove - but I'm not exactly sure why it's there, so I'll leave that for a separate change.